.pipelines/cni/lsg/kernel-upgrade-template.yaml

```diff
@@ -0,0 +1,137 @@
+parameters:
```

Comment:
Can we get someone to review to Upgrade kernel steps ?
otherwise lgtm

Reply:
I had LSG review them with me 2 weeks ago. I can get them to sign off if needed.